### ISSUE TITLE
How to run a project locally?

### ISSUE THREAD

**Issue by pixelkoduje** [Ulysseassoo/ChatMore#6]
I see that there is no sql file in the project folder to run the project locally. Can you upload it? I would like to test the project locally. Thanks

**Comment by Ulysseassoo** [comment]
I will do that tonight !

**Comment by pixelkoduje** [comment]
Great design. I'm waiting for sql file to test your code. What is your hourly rate for work by the way? And where are you from?

**Comment by pixelkoduje** [comment]
A great project. I'm waiting for sql file to test your code. What is your hourly rate for work by the way? And where are you from?

**Comment by Ulysseassoo** [comment]
Yes I'll be doing that tonight. My hourly rate for now I don't know it, it depends on the work. And I'm from France. If you want more information contact me by email

**Comment by pixelkoduje** [comment]
Sure, I'll take a look tonight

**Comment by Ulysseassoo** [comment]
Took me a while, but here you have the sql file. 

Do you need a specific tutorial on how to setup the project locally or is it okay ?
[schema.zip](https://github.com/Ulysseassoo/ChatMore/files/11990811/schema.zip)

**Comment by mahfuz0001** [comment]
do you mind giving step by step tutorial if possible

**Comment by Ulysseassoo** [comment]
Yes I'll update the readme accordingly next weekend. Stay tuned
